osu.Game.Rulesets.Catch/UI/CatcherArea.cs

```diff
@@ -190,15 +190,15 @@ public void Add(DrawableHitObject fruit)
             /// <returns>Whether the catch is possible.</returns>
             public bool AttemptCatch(CatchHitObject fruit)
             {
-                const double relative_catcher_width = CATCHER_SIZE / 2;
+                const double relative_catcher_width = CATCHER_SIZE;
 
                 // this stuff wil disappear once we move fruit to non-relative coordinate space in the future.
                 var catchObjectPosition = fruit.X * CatchPlayfield.BASE_WIDTH;
                 var catcherPosition = Position.X * CatchPlayfield.BASE_WIDTH;
 
-                var validCatch =
-                    catchObjectPosition >= catcherPosition - relative_catcher_width / 2 &&
-                    catchObjectPosition <= catcherPosition + relative_catcher_width / 2;
+                var validCatch =                                                                    //best setup I found

```

Comment:
not sure what these comments are trying to achieve

Reply:
These just for information, which combination works the best for valid catching.

Reply:
this is known and intentional. the sprite/catcher is currently sized wrongly.

Reply:
I agree, that's why I made this combination not equal